packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -130,9 +133,24 @@ async function handler(req: NextApiRequest & { userId?: number }) {
         locale: attendee.locale ?? "en",
       },
     };
-  });
+
+    // Check for the presence of hosts to determine if it is a team event type
+    if (hostsPresent) {
+      // If the attendee is a host then they are a team member
+      const teamMember = bookingToDelete.eventType?.hosts.some((host) => host.user.email === attendee.email);
+      if (teamMember) {
+        teamMembersPromises.push(attendeeObject);
```

Comment:
Why are these being considered promises?

Reply:
It's because `getTranslation` needs to run for each attendee object on line 132

Reply:
<img width="822" alt="Screenshot 2023-02-24 at 10 00 16 AM" src="https://user-images.githubusercontent.com/1780212/221092038-55d88c6e-71ea-41a4-a279-0192e8c228e0.png">
I think there is some confusion. `attendeeObject` doesn't have any promise in it. By using await in front of `getTranslation` the result is already a resolved value.